### ISSUE TITLE
Still applying code formatting?

### ISSUE THREAD

**Issue by flhorizon** [qrilka/xlsx#159]
Hello, in my other PRs I've tried to run `stylish-haskell` but it did incur massive changes over all the sources.

Would you like a PR for this?

**Comment by qrilka** [comment]
Have you seen https://github.com/qrilka/xlsx#hacking ?

**Comment by flhorizon** [comment]
Yes that's why I'm asking, because due to the amount of changes I didn't include formatting in my other MRs 😅 

Looks like it hasn't been done for "a while" :)

**Comment by qrilka** [comment]
since it was added there were not so many changes

**Comment by flhorizon** [comment]
I'll make a formatting PR after we're done with ST_State
